search/collectors/topn_test.go

```diff
@@ -398,21 +399,28 @@ func TestPaginationSameScores(t *testing.T) {
 			t.Errorf("doc ID %s is in top 5 and next 5 result sets", hit.ID)
 		}
 	}
-
 }
 
 func BenchmarkTop10of100000Scores(b *testing.B) {
-	benchHelper(10000, NewTopScorerCollector(10), b)
+	benchHelper(10000, func() search.Collector {
```

Comment:
the 10000 here doesn't seem to match the func name of "BenchmarkTop10of100000Scores"?